src/index.js

```diff
@@ -138,6 +143,9 @@ client.on(Events.GuildMemberRemove, async (member) => {
 
 client.on(Events.GuildAuditLogEntryCreate, async (auditLog) => {
     await logging.auditLogs(auditLog);
+    if (auditLog.action == AuditLogEvent.InviteCreate) {
+        invites = await client.guilds.cache.first().invites.fetch();
```

Comment:
ok im sure that the cache here works great but ideally can you not rely on the cache